staff/jakub-gaba/course-1/task-3/main.css

```diff
@@ -15,6 +15,13 @@
   align-items: center;
   justify-content: center;
   margin-top: 20em;
```

Comment:
This is a huge margin, it is really needed using flex? Same on line 20 (container_safety).